packages/react/modules/components/configure-adapter/configure-adapter.tsx

```diff
@@ -20,18 +20,14 @@ import AdapterContext, { createAdapterContext } from '../adapter-context';
 
 type valueof<T> = T[keyof T];
 
-type AdapterStates = {
-  UNCONFIGURED: string;
-  CONFIGURING: string;
-  CONFIGURED: string;
-};
-export const AdapterStates: AdapterStates = {
+export const AdapterStates = {
```

Comment:
Nit: variables should start with lowercase `adapterStates`

Reply:
Yeah, this is me being weird however. It's strange to ready `adapterStates.current = adapterStates.PENDING`. I agree that only new-able things should be capitalized. I just subjectively like it like this :D

Reply:
Ah I see. Yeah fine :)